### ISSUE TITLE
Bug: ignore exempt `<img>` elements in `require-image-title`

### ISSUE THREAD

**Issue by electrohyun** [eslint-markdown/eslint-markdown#727]
### Environment

ESLint version: v9.39.5
@eslint/markdown version: v.8.0.3
eslint-markdown version: v0.16.0
Node version: v24.11.1
npm version: v11.6.4
Operating System: Windows 11

### Which language are you using?

commonmark & gfm

### What did you do?

<details>
<summary>Configuration</summary>

```js
import { defineConfig } from 'eslint/config';
import markdown from '@eslint/markdown';
import md from 'eslint-markdown';

export default defineConfig([
  {
    files: ['repro-commonmark.md'],
    plugins: {
      markdown,
      md,
    },
    language: 'markdown/commonmark',
    rules: {
      'md/require-image-title': 'error',
    },
  },
  {
    files: ['repro-gfm.md'],
    plugins: {
      markdown,
      md,
    },
    language: 'markdown/gfm',
    rules: {
      'md/require-image-title': 'error',
    },
  },
]);

```

</details>

```md
<img id="1" />
<img src="image.png" alt="" />
<img src="image.png" aria-hidden="true" />
```

### What did you expect to happen?

The rule should ignore the following elements.

- An `<img>` element that has none of the `src`, `srcset`, or `alt` attributes.
- An `<img>` element with `alt=""`.
- An `<img>` element with `aria-hidden="true"`.

### What actually happened?

The rule reports that all three `<img>` elements are missing a title attribute.

```
~/projects/stackblitz-starters-6yjgiguu 1s
❯ npx eslint repro-commonmark.md repro-gfm.md

/home/projects/stackblitz-starters-6yjgiguu/repro-commonmark.md
  1:1  error  Images should have a title attribute  md/require-image-title
  2:1  error  Images should have a title attribute  md/require-image-title
  3:1  error  Images should have a title attribute  md/require-image-title

/home/projects/stackblitz-starters-6yjgiguu/repro-gfm.md
  1:1  error  Images should have a title attribute  md/require-image-title
  2:1  error  Images should have a title attribute  md/require-image-title
  3:1  error  Images should have a title attribute  md/require-image-title

✖ 6 problems (6 errors, 0 warnings)
```

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/stackblitz-starters-chv9dmic?file=eslint.config.mjs

### Participation

- [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [x] No - I did not use AI to generate this issue report.
- [ ] Yes - minimal assistance (e.g., IDE autocomplete, small code completions, grammar fixes).
- [ ] Yes - substantive assistance (AI-generated non-trivial portions of code, tests, or documentation).

### Additional comments

This issue follows the discussion in #719 and tracks the `require-image-title` behavior separately, as requested.

**Comment by lumirlumir** [comment]
Thanks for a separate issue!
